cruise/CruiseInformation.csv

```diff
@@ -284,4 +284,5 @@ EK20220510,Elakha,2022-05-10T16:30:00,2022-05-10T22:04:00,Endurance glider and C
 SKQ202208S,Sikuliaq,2022-05-11T00:00:00,2022-05-30T23:59:00,Papa 9
 TS20220518,Tenacious II,2022-05-18T11:00:00,2022-05-19T03:00:00,Pioneer 18a glider deployments
 TS20220526,Tenacious II,2022-05-26T01:00:00,2022-05-26T23:00:00,Pioneer 18b test glider deployments and cp_379 recovery
-AR69-01,Neil Armstrong,2022-06-20T00:00:00,2022-07-17T23:59:00,Irminger 9
\ No newline at end of file
+EK20220526,Elakha,2022-05-26T16:02:00,2022-05-26T19:04:00,Endurance CE02SHSP turn

```

Comment:
You have the launch time of the CSPP as the end time of the cruise. Time should be 22:22 from the config sheet.